source/features/list-prs-for-file.tsx

```diff
@@ -135,11 +138,7 @@ async function init(): Promise<void> {
 }
 
 async function initEditing(): Promise<void> {
-	const [path, prsByFile] = await Promise.all([
-		getCurrentPath(),
-		getPrsByFile(),
-	]);
-	let prs = prsByFile[path];
+	let prs = await getPrsForCurrentPath();
```

Comment:
`path` is used below so this can not be made into a standalone function.

This is not duplicating a lot of code so I think it's fine to keep it as is.